### ISSUE TITLE
Possible Step 3 Markdown improvement

### ISSUE THREAD

**Issue by lithafnium** [open-learning-exchange/open-learning-exchange.github.io#2463]
### Problem
Suggestion: 

On step 3, the paragraph that says: 
> ...you will be on your repository < YourUserName >/open-learning-exchange.github.io

and 
>...rename your repository to < YourUserName >.github.io. See example below.

might be confusing to some people who are new to github as they might not see the <> as place holders. 

### Steps to reproduce the problem

N/A

### Screenshots

N/A

### Proposed solution

Change to: 

> your_user_name/open-learning-exchange.github.io

and 

> your_user_name.github.io

**Comment by ChiragAwale** [comment]
Maybe adding italics to the < > is also an alternative?

**Comment by irisb1701** [comment]
I made a similar suggestion on issue #2453 , but I agree with ChiragAwale, they are already used throughout First Steps, so it might be better to just create a note reminding the reader to not include the angled brackets.

**Comment by hkcyf369** [comment]
I agree with you that the <>may be confusing to new git users. I agree with irisb1701 that just create a note reminding <> should not be included.

**Comment by hemu-codes** [comment]
Because I was also initially very confused with this, I think just providing an example link would be the easiest approach.

**Comment by lmmrssa** [comment]
I don't thin removing `<>` are required, this is general practice in programming to denote dynamic replaceable values. But yes you can change it to `code block` and also you can put note on starting somewhere saying if you see **<>** then that needs to be replaced by your equivalent value for that placeholder.

**Comment by lillyxxx** [comment]
Since it is common to use angled brackets as place holders, so I agree with @lmmrssa that there's no need to remove it. And in #2453 it shows that on Step 6 - GitHub Issues Tutorial there's notes mentioned that, but Markdown part is before Step 6. And actually, the angled brackets appear somewhere else, so maybe add a small note on the beginning of first steps will be a better solution to resolve all these similar problems.

**Comment by pattanawadee88** [comment]
I don't think removing angled brackets are necessary. I don't have any problem with <> personally.  As programmers, we all know that <> is just an indicator that we have to replace it with something else.

But adding a note would be helpful for some people who are new to programming.

**Comment by ewhang5** [comment]
I also think that removing <> is necessary. Maybe adding clarification the very first time it shows up would work, but as a programmer, I think that this format makes a lot of sense.
